plugins/modules/database/misc/redis.py

```diff
@@ -20,7 +20,7 @@
             - C(config) ensures a configuration setting on an instance.
             - C(flush) flushes all the instance or a specified db.
             - C(replica) sets a redis instance in replica or master mode. (C(slave) is an alias for C(replica).)
-        choices: [ config, flush, replica, slave ]
+        choices: [ config, flush, replica, slave, set, get, incr ]
```

Comment:
```suggestion
        choices: [ config, flush, replica, slave, set, incr ]
```
Querying information should be done in a separate `_info` module.

Reply:
I'll remove the three commands and put them into new modules. As for the `get` command I am not sure whether to put it into the existing `redis_info` module or create a new one. The existing module is more for general information about the Redis server. I think a separate module for retrieving actual key value pairs from the Redis database would make sense.